core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -445,6 +477,31 @@ private void putNull(int rowId) {
     }
   }
 
+  /**
+   * Get null at rowId
+   */
+  private Object getNull(int rowId) {
```

Comment:
It is not clear what this function return, can you add more description in comment

Reply:
This method is a supplement for the previous method `setNull`, since they are neighbours, I think it's not difficult to understand them at a time.